_rules/aria-required-owned-element-bc4a75.md

```diff
@@ -0,0 +1,269 @@
+---
+id: bc4a75
+name: ARIA required owned elements
+rule_type: atomic
+description: |
+  This rule checks that an element with an explicit semantic role has at least one of its required owned elements.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility tree
+  - DOM Tree
+acknowledgements:
+  authors:
+    - Wilco Fiers
+  previous_authros:
+    - Audrey Maniez
+    - Jey Nandakumar
+---
+
+## Applicability
+
+The rule applies to any HTML or SVG element that is [included in the accessibility tree and has a [WAI-ARIA 1.1][] [explicit semantic role][] with [required owned elements][], except if
+
+- the element has an [implicit semantic role][] that is identical to its [explicit semantic role][]; or
+- the element has a [semantic role][] of `combobox`; or
+- the element has the `aria-busy` state set to `true`, or has an [ancestor][] in the accessibility tree with this state.
+
+**Note:** An example of an element that has a [required owned element][] is `tablist` which has `tab` as a [required owned element][].
+
+**Note:** An example of an element that has an [implicit semantic role][] that is identical to its [explicit semantic role][] is a `ul` element that has `role="list"`. These elements are not applicable.
+
+**Note:** The applicability of this rule is limited to only the [WAI-ARIA 1.1][] roles, since there are unresolved issues with how [Digital Publishing WAI-ARIA Module][] (DPUB ARIA 1.1) uses role inheritance to define the [required owned element][], which makes it deviate from the model defined in [WAI-ARIA 1.1][]. The [WAI-ARIA Graphics Module][] does not include any [required owned element][].
+
+**Note:** The combobox role is excluded from this rule, because the design pattern for it as described in ARIA 1.1 has proven problematic. The combobox will be significantly different for ARIA 1.2, where it does not have [required owned elements][].
+
+## Expectation
+
+Each test target only [owns][] elements with a [semantic role][] from the [required owned element][] list for the test target's [semantic role]().
+
+**Note:** If the [required owned element][] list has two roles in one item separated by a "→", elements with the first of these roles, is only a [required owned element][] if all its owned elements have the second of these roles as their [semantic role][]. For example a `list` has `group → listitem`, so an element with the [semantic role][] of `group` only counts as a [required owned element][] for `list`, if all elements [owned by][] the `group` have the [semantic role][] of `listitem`.
```

Comment:
I understand what this means, but I had to dig into the specs to get it, and I don't think that this sentence currently makes sense…

```suggestion
**Note:** Some [required owned elements][] are only valid if they themselves [own][owns] (or "contain") elements with a given [semantic role][]. This is denoted by an arrow (meaning "containing") in the role description. For example, the role `list` has `group → listitem` as one of its [required owned elements][], meaning that elements with a role of `list` may only [owns][] elements with a role of `group` who themselves only [own][] elements with a role of `listitem`.
```

(I think it's important to use the word "contain" to denote the arrow, given that it is how ARIA calls it)